contrib/go-chi/chi.v5/chi.go

```diff
@@ -12,6 +12,8 @@ import (
 	"net/http"
 
 	"gopkg.in/DataDog/dd-trace-go.v1/contrib/internal/httptrace"
+	//MTOFF - not sure what to name this, when http and httptrace already exist
```

Comment:
**QTNA**: Is this a fair import name?

Reply:
Since other imports of `"gopkg.in/DataDog/dd-trace-go.v1/contrib/net/http"` alias to `httptrace` perhaps we should keep that consistent, and just alias `"gopkg.in/DataDog/dd-trace-go.v1/contrib/internal/httptrace"` to `httptraceinternal` instead

Reply:
That makes sense to me -- should I do that in this PR, or should we separate it?